### PR TITLE
Refine 'phantom' range slider

### LINKED ISSUE #1378: Refine "phantom" range slider
- make dark mode more consistent with light mode
- make contrast of numeric input text higher
- make contrast of track + thumb lower
- try to reduce the gap at the top of the numeric input, which causes the whole row to be taller than needed

### PR DESCRIPTION
**Pull request by ollimeier**
Fixes #1378 

Screenshot 2024-05-22 at 12 40 33

Screenshot 2024-05-22 at 12 40 56

### CODE REVIEW

File: src/fontra/client/web-components/designspace-location.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import * as html from "/core/html-utils.js";
 import { UnlitElement, htmlToElement } from "/core/html-utils.js";
 
 const colors = {
-  "disabled-color": ["#ccc", "#777"],
+  "disabled-text-color": ["#777", "#ccc"],
```
Comment (justvanrossum) [inline src/fontra/client/web-components/designspace-location.js:7]: Please undo the changes in this file. The text request was specifically about the numeric input, not the info box.

Comment (justvanrossum) [reply src/fontra/client/web-components/designspace-location.js:7]: But... please change the `gap` of `.slider-group` to `0.1em`

**Review by justvanrossum — COMMENTED** [review]
Two small requests. Otherwise looks good, thanks!

File: src/fontra/client/web-components/designspace-location.js (hunk 1 above)
Comment (ollimeier) [reply src/fontra/client/web-components/designspace-location.js]: Done.
